src/_utils/item/Item.tsx

```diff
@@ -101,6 +101,24 @@ export const Item = (props: ItemProps) => {
 
   const getTextColor = (textColor: string) => (disabled ? color.gray : textColor)
 
+  const getLeftTitle = (value: any) => {
```

Comment:
You can create a letTitle type and keep type safety (instead of using any):

```
type  leftTitleType = string | React.ReactNode
... 
readonly leftTitle?: leftTitleType
...
const getLeftTitle = (value: leftTitleType) => {
...
```